tutorials/CommandsCustomResponses.md

```diff
@@ -0,0 +1,79 @@
+> This feature is implemented in Klasa **0.5.0**, check the PR that implemented it [here](https://github.com/dirigeants/klasa/pull/162).
+
+## Custom Responses
+
+This is one of the most requested features for Klasa: the ability to translate and/or customize the error messages when a parameter is missing. This is now achievable thanks to {@link Command.customizeResponse}, which takes the name of the key of the possible (remember the {@tutorial CommandsArguments} tutorial) and either a `string` or a `function` that takes one parameter: `msg` being a `KlasaMessage` instance, that you can use to get the Language instance from (check {@link KlasaMessage.language}).
+
+## Configuring/Creating a custom response
+
+They require an argument name from the `usageString`, as you have seen in {@tutorial CommandsArguments}, the `name` is what identifies a Tag/argument, to do so, you put that name, and Klasa will modify the Tag so when it's value is not providen or is incorrect, use your custom message. Let's say we have an Overwatch command and you want Klasa to reply with a *nicer* response, for example, when you have the [RegExp](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/RegExp) type with `/(\w{3,12})#(\d{4,5})/i` and you want to notify the user that they did not write the name correctly, then you may want the message the framework sends so they can understand what's going wrong better. Consider the following command example:
+
+```javascript
+const { Command } = require('klasa');
+
+module.exports = class extends Command {
+
+	constructor(...args) {
+		super(...args, {
+			description: 'Check your Overwatch stats.',
+			usage: '<battletag:reg/(\\w{3,12})#(\\d{4,5})/i>'
+		});
+	}
+
+	async run(msg, [battletag]) {}
+
+};
+```
+
+And you want to make it so when the user doesn't input a string that matches with the RegExp, the framework sends `'You must insert a valid battletag with the format username#0000'`, then you may use the method {@link Command.customizeResponse} inside the commands' constructor:
+
```

Comment:
`And you want to make it so when the user doesn't input a ...`
doesn't sound right on tutorials and official docs.

`And you want it to send 'You must insert a valid battletag with the format username#0000' when the user doesn't input a string matching the RegExp...` is better.